### ISSUE TITLE
Horizontal header scrolling in Firefox

### ISSUE THREAD

**Issue by PieCampi** [alalonde/angular-scrollable-table#41]
First of all, thanks a lot for your component! It works beautifully with IE and Chrome, but I am having the following problem with Firefox:

when the table has more columns than its width, the headers do not scroll together with the columns contents.
Actually, I think the problem is that thedirective adds a negative margin to all the ".th-inner" elements to let the header scroll left, but in Firefox the header flees to the left, regardless of the table content.

I would fix it myself, but I've just started with AngularJS and I only managed to mess things around.

Thank you!
![header](https://cloud.githubusercontent.com/assets/8363159/14282826/77418ada-fb40-11e5-9f52-3e31d941569c.PNG)

**Comment by anthony-o** [comment]
Same problem using FF 45.0.2 on Windows, Angular 1.5.5 and angular-scrollable-table#94e9ef4dc6 .
I've with or without `<div class="table-responsive">` I've got the same result.

Here is my code.

The html part:
```html
<div class="table-responsive">
	<scrollable-table watch="list">
		<table ng-table="tableParams" class="table table-condensed table-bordered table-striped">
			<thead>
			  <tr>
				 <th ng-repeat="col in cols" sortable-header col="{{col}}">{{col}}</th>
			  </tr>
			</thead>
			<tbody>
				<tr ng-repeat="elem in list" row-id="{{ elem[cols[0]] }}">
					<td ng-repeat="col in cols">{{elem[col]}}</td>
				</tr>
			</tbody>
		</table>
	</scrollable-table>
</div>
```
The JS controller:
```js
.controller('TestCtrl', ['$scope', function($scope) {
	var nbCols = 10,
		nbLines = 300,
		list = [],
		cols = [],
		i, j;
	
	for (i = 0 ; i < nbCols ; i++) {
		cols.push('col' + i);
	}
	for (i = 0 ; i < nbLines ; i++) {
		var elem = {};
		for (j = 0 ; j < nbCols ; j++) {
			elem[cols[j]] = 'content '+i+'-'+j;
		}
		list.push(elem);
	}
	$scope.list = list;
	$scope.cols = cols;
}])
```

**Comment by alalonde** [comment]
Thanks for pointing me in the right direction.  Will be fixed in v1.1.1
